# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/gfx-hal/Cargo.toml
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/gfx-hal/Cargo.toml b/third_party/rust/gfx-hal/Cargo.toml
deleted file mode 100644
--- a/third_party/rust/gfx-hal/Cargo.toml
+++ /dev/null
@@ -1,52 +0,0 @@
-# THIS FILE IS AUTOMATICALLY GENERATED BY CARGO
-#
-# When uploading crates to the registry Cargo will automatically
-# "normalize" Cargo.toml files for maximal compatibility
-# with all versions of Cargo and also rewrite `path` dependencies
-# to registry (e.g., crates.io) dependencies
-#
-# If you believe there's an error in this file please file an
-# issue against the rust-lang/cargo repository. If you're
-# editing this file be aware that the upstream Cargo.toml
-# will likely look very different (and much more reasonable)
-
-[package]
-edition = "2018"
-name = "gfx-hal"
-version = "0.9.0"
-authors = ["The Gfx-rs Developers"]
-description = "gfx-rs hardware abstraction layer"
-homepage = "https://github.com/gfx-rs/gfx"
-documentation = "https://docs.rs/gfx-hal"
-keywords = ["graphics"]
-license = "MIT OR Apache-2.0"
-repository = "https://github.com/gfx-rs/gfx"
-
-[lib]
-name = "gfx_hal"
-path = "src/lib.rs"
-[dependencies.bitflags]
-version = "1.0"
-
-[dependencies.external_memory]
-version = "0.0.1"
-package = "external-memory"
-
-[dependencies.naga]
-version = "0.5"
-
-[dependencies.raw-window-handle]
-version = "0.3"
-
-[dependencies.serde]
-version = "1"
-features = ["serde_derive"]
-optional = true
-
-[dependencies.thiserror]
-version = "1"
-[dev-dependencies.gfx-backend-empty]
-version = "0.9"
-
-[features]
-unstable = []